lab_6_pipeline/pipeline.py

```diff
@@ -18,22 +32,49 @@ def __init__(self, path_to_raw_txt_data: Path):
         """
         Initializes CorpusManager
         """
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._validate_dataset()
+        self._storage = {}
+        self._scan_dataset()
 
     def _validate_dataset(self) -> None:
         """
         Validates folder with assets
         """
+        if not self.path_to_raw_txt_data.exists():
+            raise FileNotFoundError
+        if not self.path_to_raw_txt_data.is_dir():
+            raise NotADirectoryError
+        if not any(self.path_to_raw_txt_data.iterdir()):
+            raise  EmptyDirectoryError
+        meta_files = [i for i in self.path_to_raw_txt_data.glob('**/*.json') if  re.match(r'\d+_meta', i.stem)]
```

Comment:
there should be no other json files or txt other than text ones

Reply:
From lab desciption:

> NOTE: While validating dataset, ignore files which are named not using formats.